Format /web/media using Prettier (#24217)
mdn/content: files/en-us/web/media/formats/audio_concepts/index.md

### Joint stereo

> Normally, stereo sound is represented by audio frames which contain one sample per channel. This results in audio frames which require 2⨉ _sampleSize_ bits each, where _sampleSize_ is the number of bits each audio sample takes. Therefore, for a 16-bit stereo audio recording, each sample uses 2⨉16 or 32 bits of space. This is standard left/right (L/R) stereo or **simple stereo**.

Reviewer: ```suggestion
Stereo sound is typically represented by audio frames containing one sample per channel. This results in audio frames that require 2 times _sampleSize_ bits each, where _sampleSize_ is the number of bits each audio sample takes. Therefore, for a 16-bit stereo audio recording, each sample uses 2 times 16, or 32, bits of space. This is standard left/right (L/R) stereo or **simple stereo**.
```

Reviewer: I reworded as the ⨉ looked extremely weird.
